CHANGELOG.md

```diff
@@ -1,5 +1,90 @@
 # Changelog
 
+## [v0.4.0](https://github.com/delta-incubator/delta-kernel-rs/tree/v0.4.0/) (2024-10-22)
+
+[Full Changelog](https://github.com/delta-incubator/delta-kernel-rs/compare/v0.3.1...v0.4.0)
+
+**API Changes**
+
+*Breaking*
+
+1. `pub ScanResult.mask` field made private and only accessible as `ScanResult.raw_mask()` method [\#374]
+2. `ReaderFeatures` enum changes
```

Comment:
Can we say briefly what these are? Same for next two

Reply:
yup realized i forgot to link PRs and say more :)